secure-api-gateway-ob-uk-rs-server/src/main/java/com/forgerock/sapi/gateway/ob/uk/rs/server/api/obie/payment/v4_0_0/internationalscheduledpayments/InternationalScheduledPaymentConsentsApiController.java

```diff
@@ -0,0 +1,148 @@
+            OBValidationService<OBWriteInternationalScheduledConsent5> consentValidator,
+            ExchangeRateService exchangeRateService,
+            OBWriteInternationalScheduledConsentResponse6Factory consentResponseFactory,
+            FundsAvailabilityService fundsAvailabilityService,
+            OBWriteFundsConfirmationResponse1Factory fundsConfirmationResponseFactory) {
+
+        this.consentStoreApiClient = consentStoreApiClient;
+        this.consentValidator = consentValidator;
+        this.exchangeRateService = exchangeRateService;
+        this.consentResponseFactory = consentResponseFactory;
+        this.fundsAvailabilityService = fundsAvailabilityService;
+        this.fundsConfirmationResponseFactory = fundsConfirmationResponseFactory;
+    }
+
+    @Override
+    public ResponseEntity<OBWriteInternationalScheduledConsentResponse6> createInternationalScheduledPaymentConsents(
+            String authorization, String xIdempotencyKey, String xJwsSignature, OBWriteInternationalScheduledConsent5 obWriteInternationalScheduledConsent5, String xFapiAuthDate, String xFapiCustomerIpAddress,
+            String xFapiInteractionId, String xCustomerUserAgent, String apiClientId, HttpServletRequest request, Principal principal) throws OBErrorResponseException {
+
+        logger.info("Processing createInternationalScheduledPaymentConsents request - consent: {}, idempotencyKey: {}, apiClient: {}, x-fapi-interaction-id: {}",
+                obWriteInternationalScheduledConsent5, xIdempotencyKey, apiClientId, xFapiInteractionId);
+
+        consentValidator.validate(obWriteInternationalScheduledConsent5);
+
+        final CreateInternationalScheduledPaymentConsentRequest createRequest = new CreateInternationalScheduledPaymentConsentRequest();
+        createRequest.setConsentRequest(FRWriteInternationalScheduledConsentConverter.toFRWriteInternationalScheduledConsent(obWriteInternationalScheduledConsent5));
+        createRequest.setApiClientId(apiClientId);
+        createRequest.setIdempotencyKey(xIdempotencyKey);
+        createRequest.setCharges(calculateCharges(obWriteInternationalScheduledConsent5));
+        final OBWriteInternationalScheduled3DataInitiation initiation = obWriteInternationalScheduledConsent5.getData().getInitiation();
+        final OBWriteDomestic2DataInitiationInstructedAmount instructedAmount = initiation.getInstructedAmount();
+        createRequest.setExchangeRateInformation(exchangeRateService.calculateExchangeRateInfo(initiation.getCurrencyOfTransfer(),
+                new FRAmount(instructedAmount.getAmount(), instructedAmount.getCurrency()), initiation.getExchangeRateInformation()));
+
+        final InternationalScheduledPaymentConsent consent = consentStoreApiClient.createConsent(createRequest);
+        logger.info("Created consent - id: {}", consent.getId());
```

Comment:
Also debug, I'd expect.

Reply:
Considering what we discussed yesterday in the status call, these should remain as they are for now since ID Cloud logs will not be very helpful in this case. This information could help the debug process in sandbox environments.